tests/app/notifications/test_process_notification.py

```diff
@@ -1080,11 +1090,34 @@ def test_db_save_and_send_notification_throws_exception_deletes_notification(
 
         with pytest.raises(Boto3Error):
             db_save_and_send_notification(notification)
-        mocked.assert_called_once_with([(str(notification.id))], queue=QueueNames.SEND_SMS_MEDIUM)
+        mocked.assert_called_once_with(
+            [(str(notification.id))], queue=QueueNames.SEND_SMS_MEDIUM, retry=True, retry_policy=RETRY_POLICY_DEFAULT
+        )
 
         assert Notification.query.count() == 0
         assert NotificationHistory.query.count() == 0
 
+    @pytest.mark.parametrize(
+        ("notification_type, process_type, expected_retry, expected_retry_period"),
+        [
+            (EMAIL_TYPE, BULK, 48, 300),
+            (EMAIL_TYPE, NORMAL, 48, 300),
+            (EMAIL_TYPE, PRIORITY, 48, 300),
+            (SMS_TYPE, BULK, 48, 300),
+            (SMS_TYPE, NORMAL, 48, 300),
+            (SMS_TYPE, PRIORITY, 48, 26),
```

Comment:
instead of hard coding these values we could pull them from `RETRY_POLICY_DEFAULT` and `RETRY_POLICY_HIGH`

Reply:
Changed it to the config values. 👍